.schema/openapi.json

```diff
@@ -2006,6 +2006,81 @@
         ]
       }
     },
+    "/self-service/saml/idp":{
+      "get":{
+        "description": "This endpoint is to redirect the user to the idp auth flow",
```

Comment:
All other descriptions are something like "This endpoint + verb at the active form".